internal/cli/atlas/atlas_darwin.go

```diff
@@ -16,4 +16,4 @@
 
 package atlas
 
-const deprecatedMessage = "\u26A0 There's a new, dedicated Atlas CLI available for Atlas users. Install the Atlas CLI to enjoy the same capabilities and keep getting new features: https://dochub.mongodb.org/core/migrate-to-atlas-cli. Atlas commands for MongoCLI are now deprecated, but you can keep using them for 12 months (until April 30, 2023).\n\n"
+const deprecatedMessage = "\u26A0 There's a new, dedicated Atlas CLI available for Atlas users. Install the Atlas CLI to enjoy the same capabilities and keep getting new features. Run \"brew install mongodb-atlas-cli\" or visit https://dochub.mongodb.org/core/migrate-to-atlas-cli. Atlas commands for MongoCLI are now deprecated, but you can keep using them for 12 months (until April 30, 2023).\n\n"
```

Comment:
Regarding the phrase: "now deprecated", can we change this to say: "are deprecated since March 30, 2022, but you can keep ...." -- or whatever date? I don't know the exact date, so please use another date if you know it.

Reply:
Hi Julia, this message was suggested by the product team and the doc team. I will send you a private message to discuss this. I will open a follow-up PR to update this message if needed. Thanks!